packages/bun-usockets/src/crypto/openssl.c

```diff
@@ -44,1637 +44,1892 @@ void* sni_find(void* sni, const char* hostname);
-    }
+long BIO_s_custom_ctrl(BIO *bio, int cmd, long num, void *user) {
+  switch (cmd) {
+  case BIO_CTRL_FLUSH:
+    return 1;
+  default:
+    return 0;
+  }
 }
 
-int BIO_s_custom_write(BIO* bio, const char* data, int length)
-{
-    struct loop_ssl_data* loop_ssl_data = (struct loop_ssl_data*)BIO_get_data(bio);
+int BIO_s_custom_write(BIO *bio, const char *data, int length) {
+  struct loop_ssl_data *loop_ssl_data =
+      (struct loop_ssl_data *)BIO_get_data(bio);
 
-    loop_ssl_data->last_write_was_msg_more = loop_ssl_data->msg_more || length == 16413;
-    int written = us_socket_write(0, loop_ssl_data->ssl_socket, data, length, loop_ssl_data->last_write_was_msg_more);
+  loop_ssl_data->last_write_was_msg_more =
+      loop_ssl_data->msg_more || length == 16413;
+  int written = us_socket_write(0, loop_ssl_data->ssl_socket, data, length,
+                                loop_ssl_data->last_write_was_msg_more);
 
-    if (!written) {
-        BIO_set_flags(bio, BIO_FLAGS_SHOULD_RETRY | BIO_FLAGS_WRITE);
-        return -1;
-    }
+  if (!written) {
+    BIO_set_flags(bio, BIO_FLAGS_SHOULD_RETRY | BIO_FLAGS_WRITE);
+    return -1;
+  }
 
-    // printf("BIO_s_custom_write returns: %d\n", written);
+  // printf("BIO_s_custom_write returns: %d\n", written);
 
-    return written;
+  return written;
 }
 
-int BIO_s_custom_read(BIO* bio, char* dst, int length)
-{
-    struct loop_ssl_data* loop_ssl_data = (struct loop_ssl_data*)BIO_get_data(bio);
+int BIO_s_custom_read(BIO *bio, char *dst, int length) {
+  struct loop_ssl_data *loop_ssl_data =
+      (struct loop_ssl_data *)BIO_get_data(bio);
 
-    // printf("BIO_s_custom_read\n");
+  // printf("BIO_s_custom_read\n");
 
-    if (!loop_ssl_data->ssl_read_input_length) {
-        BIO_set_flags(bio, BIO_FLAGS_SHOULD_RETRY | BIO_FLAGS_READ);
-        return -1;
-    }
+  if (!loop_ssl_data->ssl_read_input_length) {
+    BIO_set_flags(bio, BIO_FLAGS_SHOULD_RETRY | BIO_FLAGS_READ);
+    return -1;
+  }
 
-    if ((unsigned int)length > loop_ssl_data->ssl_read_input_length) {
-        length = loop_ssl_data->ssl_read_input_length;
-    }
+  if ((unsigned int)length > loop_ssl_data->ssl_read_input_length) {
+    length = loop_ssl_data->ssl_read_input_length;
+  }
 
-    memcpy(dst, loop_ssl_data->ssl_read_input + loop_ssl_data->ssl_read_input_offset, length);
+  memcpy(dst,
+         loop_ssl_data->ssl_read_input + loop_ssl_data->ssl_read_input_offset,
+         length);
 
-    loop_ssl_data->ssl_read_input_offset += length;
-    loop_ssl_data->ssl_read_input_length -= length;
-    return length;
+  loop_ssl_data->ssl_read_input_offset += length;
+  loop_ssl_data->ssl_read_input_length -= length;
+  return length;
 }
 
-struct us_internal_ssl_socket_t* ssl_on_open(struct us_internal_ssl_socket_t* s, int is_client, char* ip, int ip_length)
-{
+struct us_internal_ssl_socket_t *ssl_on_open(struct us_internal_ssl_socket_t *s,
+                                             int is_client, char *ip,
```

Comment:
are these formatting choices intentional?

Reply:
was clang-format-16 auto formating not intentional